_posts/2018-03-03-neo4j-graph-database.md

```diff
@@ -0,0 +1,111 @@
+---
+layout: post
+title: Neo4j, a graph database
+date: 2018-03-03
+---
+
+I went to a very cool meetup this week about [Neo4j](https://neo4j.com/), a graph database. It is not something I have had the chance to get my hands on yet, so it was great to get a taste of it. This post is a short introduction to graph database and a small example of what one can do with Neo4j.
+
+### Graph databases
+
+##### Definition
+
+> In computing, a graph database is a database that uses graph structures for semantic queries with nodes, edges and properties to represent and store data. A key concept of the system is the graph (or edge or relationship), which directly relates data items in the store.
+> [Wikipedia](https://en.wikipedia.org/wiki/Graph_database)
+
+So a graph database uses a graph at its core, this is very important, and to insist even more on the idea of graph:
```

Comment:
The last part of this sentence sounds a bit clunky. Perhaps something like:

So a graph database uses a graph at its core, ~this is very important, and to insist even more on the idea of graph:~ and to emphasise its importance :